societe-archeologique-de-bordeaux.csl

```diff
@@ -0,0 +1,304 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" class="note" demote-non-dropping-particle="never" default-locale="fr-FR" page-range-format="expanded">
+  <info>
+    <title>Société Archéologique de Bordeaux</title>
+	    <title-short>SAB</title-short>
+    <id>http://www.zotero.org/styles/societe-archeologique-de-bordeaux</id>
+    <link href="http://www.zotero.org/styles/societe-archeologique-de-bordeaux" rel="self"/>
+    <link href="http://www.societe-archeologique-bordeaux.fr/publications/recommandations-aux-auteurs.html" rel="documentation"/>
+    <author>
+      <name>Hélène Réveillas</name>
+      <email>hreveillas@bordeaux-metropole.fr</email>
+    </author>
+    <category citation-format="note"/>
+    <category field="history"/>
+    <summary>Style reprenant les normes diffusées par la SAB</summary>
+    <updated>2016-11-09T08:31:00+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="fr">
+    <terms>
+      <term name="editor">
+        <single>(dir.) </single>
+        <multiple>(dir.)</multiple>
+      </term>
+      <term name="editortranslator">
+        <single>(éd.)</single>
+        <multiple>(éd.)</multiple>
+      </term>
+      <term name="in">dans</term>
+      <term name="anonymous">anonyme</term>
+      <term name="anonymous" form="short">anon.</term>
+      <term name="accessed">consulté le</term>
+      <term name="no date">sans date</term>
+      <term name="no date" form="short">s.&#160;d.</term>
+      <term name="sub verbo">s.u.</term>
+    </terms>
+  </locale>
+  <macro name="author">
+    <names variable="author" delimiter=", ">
+      <name and="text" name-as-sort-order="all" sort-separator=", " delimiter-precedes-last="never" delimiter=", "/>
+      <substitute>
+        <text macro="editor"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="editor">
+    <names variable="editor">
+      <name and="text" name-as-sort-order="all" sort-separator=", " delimiter-precedes-last="never" delimiter=", "/>
+      <label form="short" text-case="lowercase" prefix=", "/>
+    </names>
+  </macro>
+  <macro name="pages">
+    <group>
```

Comment:
> no it doesn't work. I don't have the pages with that

it definitely does work. Did you replace the `<text macro="pages"/>` with `<text variable="page"/>`?